src/constraints.jl

```diff
@@ -682,6 +682,46 @@ function set_normalized_coefficient(
     return
 end
 
+"""
+    set_normalized_coefficients(
+        con_ref::ConstraintRef,
+        variable,
+        new_coefficients::Vector{Tuple{Int64,T}},
+    )
+
+Set the coefficients of `variable` in the constraint `con_ref` to
+`new_coefficients`.
+
+Note that prior to this step, during constraint creation, JuMP will aggregate
+multiple terms containing the same variable.
+
+```jldoctest; setup = :(using JuMP), filter=r"≤|<="
+model = Model()
+@variable(model, x)
+@constraint(model, con, [2x + 3x, 4x] in MOI.Nonnegatives(2))
+set_normalized_coefficients(con, x, [(1, 2.0), (2, 5.0)])
```

Comment:
One design decision is whether this is the nicest design. Another option might be `[2.0, 5.0]`, and you have to provide explicit zeros.

Reply:
Let's keep close to the MOI input.